_pages/algorithm-development/algorithm-checklist.md

```diff
@@ -99,7 +99,7 @@ A common gotcha for setting and updating the sample input is that you must publi
 
 ### Update Pricing:
 
-If you are publishing a new version of your algorithm, you can update the pricing of each call. This will be displayed on the pricing section of your algorithm profile. Remember, you can set the royalty price even if your code is open source!
+If you are publishing a new version of your algorithm, you can update the pricing of each call. This will be displayed next to your algorithm's name on the overview page. Remember, you can set the royalty price even if your code is open source!
 
 <img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algorithm_checklist/cost.png" alt="inputs and outputs" class="screenshot img-md">
```

Comment:
⚠️ Looks like this image was deleted below - do we need to delete this line too?

Reply:
It's not deleted, I just changed it to a different image?

Reply:
[This one](https://github.com/algorithmiaio/dev-center/blob/f41066e3350188cf69b9b11e2c18aca8611dd448/images/post_images/algorithm_checklist/cost.png) to be precise

Reply:
Brilliant LGTM then! Thx!

Reply:
Ah, GitHub's UX threw me a bit. Showed me a big negative red on the bytes for the image, but if I toggle the rich viewer I see the new version. Excellent!